projects/js-packages/licensing/jest.config.cjs

```diff
@@ -2,4 +2,11 @@ const baseConfig = require( 'jetpack-js-tools/jest/config.base.js' );
 
 module.exports = {
 	...baseConfig,
+	setupFiles: [ ...baseConfig.setupFiles, '<rootDir>/jest-globals.js' ],
+	transform: {
```

Comment:
This is needed otherwise mocking is not working properly because of the es6 modules